temp.c

```diff
@@ -173,65 +259,22 @@ void temp_sensor_tick() {
 
 				#ifdef	TEMP_THERMISTOR
 				case TT_THERMISTOR:
-					do {
-						uint8_t j, table_num;
-						//Read current temperature
-						temp = analog_read(i);
```

Comment:
Yes. You're right. I read analog part and somehow I thought that i reads pin X. Reverted to analog_read(i)